From 00e1113a2ecb8ea1e35c2069b5eec1635a83f804 Mon Sep 17 00:00:00 2001
From: Alessandro Astone <ales.astone@gmail.com>
Date: Sat, 4 May 2019 20:06:48 +0200
Subject: [PATCH 1/2] Revert "recovery: wipe bootloader message from index 0
 when using custom offsets"

This reverts commit fbf84b52b4d47d795e2079cb41a824fc868aaaae.
---
 bootloader_message/bootloader_message.cpp | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/bootloader_message/bootloader_message.cpp b/bootloader_message/bootloader_message.cpp
index d4eae854..b70d54e5 100644
--- a/bootloader_message/bootloader_message.cpp
+++ b/bootloader_message/bootloader_message.cpp
@@ -174,11 +174,6 @@ bool write_bootloader_message(const bootloader_message& boot, std::string* err)
 
 bool clear_bootloader_message(std::string* err) {
   bootloader_message boot = {};
-  if (BOOTLOADER_MESSAGE_OFFSET_IN_MISC < sizeof(bootloader_message)) {
-    std::string misc_blk_device = get_misc_blk_device(err);
-    if (misc_blk_device.empty()) return false;
-    return write_misc_partition(&boot, sizeof(boot), misc_blk_device, 0 /* offset */, err);
-  }
   return write_bootloader_message(boot, err);
 }
 
-- 
2.17.1

